### ISSUE TITLE
Apache mpm_prefork / http2

### ISSUE THREAD

**Issue by tpnsolutions** [virtualmin/virtualmin-gpl#501]
Apache by default is setup to use "mpm_prefork" however, when you enable "http2" protocol support, you need to disable "mpm_prefork" and enable "mpm_worker" instead for "http2" to work correctly with Apache.

I happened to be checking the Apache error_log (/var/log/apache2/error_log) today and noticed that "http2" was giving an error upon Apache restart. It wasn't causing the server to fail, but simply caused "http2" to ignored. A bit of digging into the matter on the Apache website found the above suggestion which resolved things.

**Comment by iliajie** [comment]
Hello, Peter.

`mpm_prefork` is no longer the default for Apache, `mpm_event` is and it works just fine with `h2` module as far as I know.

**Comment by tpnsolutions** [comment]
I did a recent install of Virtualmin on Ubuntu 22.04 and it had "mpm_prefork" setup by default. I also had during post configuration setup "http2" to be enabled by default for all websites. As mentioned I happened to be looking at the Apache error_log and noticed that it was complaining that while "http2" was enabled, it wasn't being used due to incompatibility with "mpm_prefork". I then disabled and enabled "mpm_work" which resolved the issue.

Hence why I created this issue.
